src/multi_site_orchestrator.js

```
@@ -0,0 +1,459 @@
+     */
+    canExecuteSite(siteName) {
+        const breaker = this.circuitBreakers.get(siteName);
+        if (!breaker) return true;
+
+        switch (breaker.state) {
+            case CircuitState.CLOSED:
+                return true;
+                
+            case CircuitState.OPEN:
+                // Check if enough time has passed to try half-open
+                const timeSinceFailure = Date.now() - breaker.lastFailureTime;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)